### PR TITLE
Direct link to ./CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by jaoleal**
For convenience.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -65,7 +65,7 @@ questions or ideas you want to discuss please join us in
 [#bitcoin-rust](https://web.libera.chat/?channel=#bitcoin-rust) on
 [libera.chat](https://libera.chat).
 
-For more information please see `./CONTRIBUTING.md`.
+For more information please see [`./CONTRIBUTING.md`](./CONTRIBUTING.md).
```
Comment (storopoli) [inline README.md:68]: Maybe?

```suggestion
For more information please see [`CONTRIBUTING.md`](./CONTRIBUTING.md).
```

**Comment by tcharding** [comment]
Can you squash and force push please.

**Comment by jaoleal** [comment]
Done

**Review by tcharding — APPROVED** [review]
ACK 82a0656b57cad0410685cc7da17a163d4e7bf9c9

**Review by Kixunil — APPROVED** [review]
ACK 82a0656b57cad0410685cc7da17a163d4e7bf9c9

**Pull request merged**